magstock/templates/regextra.html

```diff
@@ -0,0 +1,116 @@
+
+<script type="text/javascript">
+    if (window.BADGE_TYPES) {
```

Comment:
The ``BADGE_TYPES`` global Javascript object on the main prereg form is how we affect the badge selection overhead row at the top of that form.  We can add to the array or modify the existing entries or remove what's already there as desired to make changes.